whisper/whisperv5/envelope.go

```diff
@@ -62,7 +62,7 @@ func (e *Envelope) rlpWithoutNonce() []byte {
 
 // NewEnvelope wraps a Whisper message with expiration and destination data
 // included into an envelope for network forwarding.
-func NewEnvelope(ttl uint32, topic TopicType, aesNonce []byte, msg *SentMessage) *Envelope {
+func NewEnvelope(ttl uint32, topic TopicType, aesNonce []byte, msg *sentMessage) *Envelope {
```

Comment:
Making sentMessage private means that no code outside package whisperv5 can call NewEnvelope. Not sure if that's good or not.